lib/mapper/product_mapper.rb

```diff
@@ -30,7 +30,7 @@ def self.map_payload(product, additional_data = {}, track_inventory = true, defa
                   type: isDigital ? 'digital' : 'physical',
                   weight: product['weight'] ? product['weight']['value'] : '0',
                   width: product['width'] ? product['width']['value'] : '0',
-                  inventory_tracking: isDigital || !track_inventory ? 'none' : 'product',
+                  inventory_tracking: isDigital || !track_inventory || !product['trackInventory'] ? 'none' : 'product',
```

Comment:
@Jhalter5Stones You should use the boolify util here around `product['trackInventory']` Due to the agents in the flow, that value may come in as `"false"` which is truthy as it's a non-blank string

Reply:
@mrogers-5s How would it come through as a string? I tested a payload through the flow and it came through as a boolean and not a string.

Reply:
@mrogers-5s Ok can you take a look and see if this looks better?

Reply:
@mrogers-5s I made updates to this. Can you take a look?